data/json/recipes/recipe_ammo.json

```diff
@@ -950,5 +950,17 @@
         [ "willowbark", 1 ]
       ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "cac2powder",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "60 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "coal_lump", 30 ], [ "quicklime", 44 ] ] ]
```

Comment:
This needs to be `[ [ [ "coal_lump", 30 ] ], [ [ "quicklime", 44 ] ] ]`, two more brackets. Now I'm getting "coal OR quicklime"

Reply:
Also, it's `material_quicklime`

Reply:
Thanks for the catch!